### PR TITLE
Add guidance link

### LINKED ISSUE #254: Mentor notes has temporary text
This is seen in all mentor requests and is not from a specific exercise.
Screen Shot 2021-08-26 at 12 14 01 PM

### PR DESCRIPTION
**Pull request by iHiD**
@kntsoriano See the two TODOs for you here.

- Firstly, `exercise.slug` doesn't seem to be set, but `exercise.id` is set (which is unexpected)
- Secondly, the URL should probably come from Rails in the normal way you do things.

The tests will also need fixing.

---

Fixes https://github.com/exercism/v3-beta/issues/254

### CODE REVIEW

**Comment by kntsoriano** [comment]
@iHiD This should be ready.

**Comment by iHiD** [comment]
@kntsoriano Tests fail and needs rebasing.

**Comment by iHiD** [comment]
@kntsoriano Think you missed this one :)

File: app/models/exercise.rb [hunk 1]
```diff
@@ -93,6 +93,8 @@ def download_cmd
     "exercism download --exercise=#{slug} --track=#{track.slug}".freeze
   end
 
+  def notes; end
```
Comment (iHiD) [inline app/models/exercise.rb]: Is this left over accidentally? I can't see it being used anywhere.

File: app/models/exercise.rb [hunk 2]
```diff
@@ -108,6 +110,10 @@ def icon_url
     "#{Exercism.config.website_icons_host}/exercises/#{icon_name}.svg"
   end
 
+  def mentoring_notes_url
```
Comment (iHiD) [inline app/models/exercise.rb]: Nice refactor :)

```suggestion
  def edit_mentoring_notes_url
```

(as this takes you directly into GH's edit interface)

Comment (kntsoriano) [reply app/models/exercise.rb]: Thanks!

File: app/models/exercise.rb (hunk 1 above)
Comment (kntsoriano) [reply app/models/exercise.rb]: Yeah. I forgot to remove it.

File: app/css/components/mentor/discussion.css [hunk 3]
```diff
@@ -511,6 +511,9 @@
         }
         & #panel-guidance {
             @apply py-16 px-24;
+
+            & .notes-cta {
+            }
```
Comment (iHiD) [inline app/css/components/mentor/discussion.css]:
```suggestion
```

**Pull request merged**